2019Labs/CustomSecurityContent/documentation/lab4_ansible.adoc

```diff
@@ -158,6 +172,25 @@ Now, we will add an Ansible task or tasks for this rule below the header in `sha
       line: "TMOUT=1800"
 ----
 
+Whole file at this moment should look like this:
+
+----
+# platform = multi_platform_all
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+- name: configure timeout
+  lineinfile:
+      create: yes
+      dest: /etc/profile
+      regexp: "^#?TMOUT"
+      line: "TMOUT=1800"
+----
+
```

Comment:
Is it now time to close the file?

And could you please add the "Use Ctrl+Shift+V" to paste into the editor hint?

Reply:
It's written few lines before the snippet. Pasting hint added